priv/gettext/zh_Hant/LC_MESSAGES/default.po

```diff
@@ -0,0 +1,478 @@
+# # `msgid`s in this file come from POT (.pot) files.
+# #
+# # Do not add, change, or remove `msgid`s manually here as
+# # they're tied to the ones in the corresponding POT file
+# # (with the same domain).
+# #
+# # Use `mix gettext.extract --merge` or `mix gettext.merge`
+# # to merge POT files into PO files.
+msgid ""
+msgstr ""
+"Language: zh_Hant\n"
```

Comment:
Here, an An IETF BCP 47 language tag is required, which is why I had to replace it with a tag from the [IANA Language Subtag Registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry). The CLDR library we use only supports a subset: https://github.com/elixir-cldr/cldr/tree/master/priv/cldr/locales Hence "zh_Hant". Hope that's okay :) The same applies to the folder name.

Reply:
Thanks~